### DISCUSSION TITLE
Ability to add 'For X minutes' for online/offline monitor inline with current implementation of other monitors

### DISCUSSION THREAD

**Discussion by dimatx** [henrygd/beszel#215] [Ideas]
Would be great if you could select to be notified of up/down (offline/online) status change based on a time-window. It blips here and there when monitoring Wi-Fi-based / low-powered devices like Raspberry Pi Zero that I'd prefer to ignore unless it goes offline for X minutes.

**Comment by henrygd** [comment]
Makes sense. I'll add it to the list but may be a while until I can get around to it.

**Comment by harupong** [comment]
+1 for this. One of my agents is on a cheap and unstable VPS, and although the server is up all the time, status alerts are sent periodically.

@henrygd Would you mind tracking this on https://github.com/henrygd/beszel/projects ?

**Reply by henrygd** [reply]
Yes, added.
